integration-tests/tests/src/tests/objects.ts

```diff
@@ -1335,4 +1335,38 @@ describe("Realm.Object", () => {
       expect(objFromKey?.age).equals(7);
     });
   });
+
+  // from https://github.com/realm/realm-js/issues/6129
+  describe("Primary key and property with default value", () => {
+    const PrimaryAndDefaultSchema: Realm.ObjectSchema = {
+      name: "MySchema",
+      primaryKey: "id",
+      properties: {
+        id: { type: "int" },
+        fieldOne: { type: "string" },
+        fieldTwo: { type: "string", default: "DEFAULT_VALUE" },
+      },
+    };
+    openRealmBeforeEach({ schema: [PrimaryAndDefaultSchema] });
+
+    [Realm.UpdateMode.All, Realm.UpdateMode.Modified].forEach((updateMode) => {
+      it(`Update properties (updateMode = ${updateMode})`, async function (this: Mocha.Context & RealmContext) {
```

Comment:
Minor suggestion to make the output read better.

```suggestion
      it(`updates properties (updateMode = ${updateMode})`, async function (this: Mocha.Context & RealmContext) {
```